packages/components/pie-button/src/button.scss

```diff
@@ -50,31 +50,31 @@ $breakpoint-wide: 768px;
             --btn-padding: 6px var(--dt-spacing-b);
             --btn-font-size: #{p.font-size(--dt-font-size-14)};
             --btn-line-height: calc(var(--dt-font-size-14-line-height) * 1px);
-            --btn-icon-size: 16px;
+            --icon-size-override: 16px;
         } @else if $size == 'small-expressive' {
             --btn-height: var(--btn-height--small);
             --btn-padding: 6px var(--dt-spacing-d);
             --btn-font-size: #{p.font-size(--dt-font-size-20)};
             --btn-line-height: calc(var(--dt-font-size-20-line-height) * 1px);
-            --btn-icon-size: 20px;
+            --icon-size-override: 20px;
         } @else if $size == 'small-productive' {
             --btn-height: var(--btn-height--small);
             --btn-padding: 8px var(--dt-spacing-d);
             --btn-font-size: #{p.font-size(--dt-font-size-16)};
             --btn-line-height: calc(var(--dt-font-size-16-line-height) * 1px);
-            --btn-icon-size: 20px;
+            --icon-size-override: 20px;
         } @else if $size == 'medium' {
             --btn-height: var(--btn-height--medium);
             --btn-padding: 10px var(--dt-spacing-e);
             --btn-font-size: #{p.font-size(--dt-font-size-20)};
             --btn-line-height: calc(var(--dt-font-size-20-line-height) * 1px);
-            --btn-icon-size: 24px;
+            --icon-size-override: 24px;
```

Comment:
@dandel10n please can you add a /snapit comment to this PR and test the snapshot package in PIE Aperture vanilla? Just want to check that this doesn't have any weird impact on it.

Happy to help with any of this if you aren't familiar with the process

Reply:
Tested locally in pie aperture vanilla js and next js apps. Checked icon button, button, modal and cookie banner. I also created [a draft PR for pie-aperture](https://github.com/justeattakeaway/pie-aperture/pull/43) with dev package versions.